spec/features/events_spec.rb

```diff
@@ -72,8 +74,21 @@
 
     click_link "Death Star's destruction celebration"
     page.should have_link('Email')
+    page.should have_link('Participants')
 
     click_link 'Email'
     page.should have_content "Emails for Death Star's destruction celebration"
+
+    visit event_path(events(:party))
+    click_link 'Participants'
+    page.should have_content "Participants of Death Star's destruction celebration"
+    page.all('.table tr').count.should == 7
+
+    page.should_not have_content 'john.skywalker@rebel-alliance.org' # User with a shipment request
```

Comment:
@bgeuken I have added a shipment request and defined this so that a tsp member is not able to see the user of a shipment request.